src/pages/nodes/start-here/setup.mdx

```diff
@@ -118,63 +124,53 @@ docs](https://docs.cosmos.network/v0.47/build/tooling/cosmovisor):
 ### From Source
 
 ```bash
-go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@latest
```

Comment:
Even the the [release says latest](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.7.1), go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.7.1 should work I think?